pom.xml

```diff
@@ -440,6 +440,7 @@ limitations under the License.
                             <exclude>**/*.txt</exclude>
                             <exclude>**/pom.xml.tag</exclude>
                             <exclude>**/pom.xml.releaseBackup</exclude>
+                            <exclude>**/*.options</exclude>
```

Comment:
Another option could be to add mapping like a few lines above:

```
<mapping>
  <options>SCRIPT_STYLE</options>
</mapping>
```

The current solution you have made is to add a header and then ignore the file from the check. I would prefer that we add the correct style for the file instead.

Reply:
Right, I wasn't really sure what I was doing. Your solution proposal makes more sense. Fixed.

Reply:
As we agreed in private conversation I reverted to excluding .options files and removed the copyright header.